doc/source/examples/incompressible-flow/2d-naca0012-low-reynolds/2d-naca0012-low-reynolds.rst

```diff
@@ -0,0 +1,291 @@
+
+.. code-block:: text
+	
+    subsection mesh
+      set type      = gmsh
+      set file name = naca.msh
+    end
+
+Below is the whole mesh and a zoom on the airfoil, for an angle of attack :math:`\alpha = 15°`
+
+.. image:: image/whole_mesh.png
+
+.. image:: image/zoomed_mesh_15.png
+
+Mesh adaptation 
+~~~~~~~~~~~~~~~~~~
+
+Mesh adaptation is used to get a higher resolution in areas of interest, that is to say close to the airfoil, while keeping a coarse mesh far from the NACA. Since the mesh is big and the simulation lengthy in time, it was chosen not to refine too much. Also, since the area of interest of the mesh (close to the airfoil) is much smaller than the whole mesh, the coarsening fraction was set eight times bigger than the refinement fraction. The parameters were tuned as follows: 
+
+.. code-block:: text
+	
+    subsection mesh adaptation
+      set type                 = kelly
+      set variable             = velocity
+      set fraction type        = number
+      set max number elements  = 700000
+      set max refinement level = 2
+      set min refinement level = 0
+      set frequency            = 5
+      set fraction refinement  = 0.02
+      set fraction coarsening  = 0.16
+    end
+    
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~
+
+The boundary conditions are defined as presented above : 
+
+.. code-block:: text
+
+     subsection boundary conditions
+       set number = 4
+       subsection bc 0
+         set type = noslip
+       end
+       subsection bc 1
+         set type = function
+         subsection u
+           set Function expression = 1
+         end
+         subsection v
+           set Function expression = 0
+         end
+         subsection w
+           set Function expression = 0
+         end
+       end
+       subsection bc 2
+         set type = slip
+       end
+       subsection bc 3
+       	set type = outlet
+       	set beta = 1.3
+       end
+     end
+	
+The boundary 0, corresponding to the NACA wall, is a ``noslip`` boundary condition which sets the velocity to zero on the boundary. The boundary 1 is the inlet where the velocity field was chosen to be horizontal and unitary, to ensure that :math:`Re = 1000` is correct. It is represented in green on the figure. The boundary 2, in black on the image, correspond to the upper and lower walls are endowed with a ``slip`` boundary condition and finally boundary 3 is of type ``outlet`` with a parameter :math:`\beta = 1.3`. The reader is referred to the `Parameters Guide <https://lethe-cfd.github.io/lethe/parameters/cfd/linear_solver_control.html>`_ for more information about the :math:`\beta` parameter.
+
+Non-linear solver
+~~~~~~~~~~~~~~~~~
+
+The ``inexact_newton`` non-linear solver is used with a high ``tolerance``, since convergence can be hard to obtain for high Reynolds number. The choice of this solver was to reduce the cost of the simulation since it reuses the Jacobian matrix between iterations.
```

Comment:
```suggestion
The ``inexact_newton`` non-linear solver is used with a high ``tolerance``, since convergence can be hard to obtain for high Reynolds numbers. The choice of this solver was to reduce the cost of the simulation since it reuses the Jacobian matrix between iterations.
```